### DISCUSSION TITLE
Captcha function

### DISCUSSION THREAD

**Discussion by wu-clan** [fastapi-practices/fastapi_best_architecture#102] [Ideas]
- [x] Login interface graphic captcha

**Comment by wu-clan** [comment]
Hi, @downdawn 

Do you have any ideas?

**Reply by downdawn** [reply]
Generally, open-source graphic verification codes are sufficient for account login, but now the more popular ones are non-inductive verification codes, which need to be connected to third parties, such as Geetest, Yidun, Dingxiang, etc.

**Reply by wu-clan** [reply]
I just released a new version, which now supports base64 strings. https://github.com/wu-clan/fast_captcha

**Comment by wu-clan** [comment]
Implementation：#123
